homeassistant/components/goodwe/select.py

```diff
@@ -15,17 +15,26 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+_OPERATION_MODES: dict[OperationMode, str] = {
+    OperationMode.GENERAL: "General mode",
```

Comment:
The options in the state machine should be lowercase snake_case and preferably have a translation via a `translation_key`.

Example:

https://github.com/home-assistant/core/blob/32a62801390586178f7693b312c8f5bac3f86c74/homeassistant/components/plugwise/select.py#L51

https://github.com/home-assistant/core/blob/32a62801390586178f7693b312c8f5bac3f86c74/homeassistant/components/plugwise/strings.json#L44-L62

Note that changing the options to snake_case in the state machine is a breaking change.

Reply:
Done, the states are snake_case and localized now.